src/python/pants/option/global_options.py

```diff
@@ -46,10 +47,21 @@ class GlobMatchErrorBehavior(Enum):
 class FileNotFoundBehavior(Enum):
     """What to do when globs do not match in BUILD files."""
 
+    ignore = "ignore"
     warn = "warn"
     error = "error"
 
     def to_glob_match_error_behavior(self) -> GlobMatchErrorBehavior:
+        deprecated_conditional(
+            lambda: self == self.ignore,
+            removal_version='1.28.0.dev2',
+            entity_description='--files-not-found-behavior-option=ignore',
+            hint_message="If you currently set `--files-not-found-behavior=ignore`, you will "
```

Comment:
Btw you can make Black make this formatting be much prettier if you use parentheses around the strings:

```
hint_message=(
    "foo"
    "bar"
)
```

Not necessary to do, but makes things cleaner.